.gitlab/quartz.yml

```diff
@@ -22,10 +22,6 @@
     MAKE_PAR: 6
     BASELINE_PAR: 18
 
-# Setup
-setup_quartz:
-  extends: [.setup, .on_quartz]
-
```

Comment:
Why was this removed?

Reply:
That’s a good question:
For now there is only a setup job for the part of the pipeline running on quartz (not for lassen).
I simplified it since it was unnecessarily convoluted. But it wasn’t a critical change.
In fact I may have to revert this change at some point when adding dependencies build on lassen...